__tests__/app.test.js

```diff
@@ -177,8 +179,53 @@ describe("app", () => {
         .get("/api/wrongpath")
         .expect(404)
     });
+
+    test("200 - can filter articles by given topic query.", () => {
+      return request(app)
+        .get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.article;
+          
+          expect(Object.keys(articles[0]).length).toBe(8);
+          
+          // Checking to have just the mitch articles after query
+          
+          articles.forEach((article) => {
+            
+          expect(article.topic).toBe("mitch");
+          });
+
+          // Checking that the articles are sorted by date in descending order
+
+            expect(body.article).toBeSortedBy('created_at', {descending: true})
+        });
+    });
+
```

Comment:
Good test 👍  I would just add a length check for the whole articles array. How many articles are you expecting with the topic of mitch?

Reply:
Thanks added :)